### PR TITLE
Hotfix/trivy ignore

### PR DESCRIPTION
**Pull request by salyateem13**
# Description

Ignoring unfixed CVEs in Trivy scan

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested by pointing actions-python to this branch, and lazy-api to branch of actions-python.

```yaml
- Test case: New unfixed CVEs are ignored
   Steps: |
     Running this (workflow)[https://github.com/variant-inc/lazy-api/runs/4940650268?check_suite_focus=true] show the following 
CVEs found by Ralph are ignored -CVE-2022-23218, CVE-2022-23219. New CVEs were previously not ignored 
      in this (workflow)[https://github.com/variant-inc/lazy-api/runs/4924915554?check_suite_focus=true].
You can also see message shows when no vulnerabilities with fixes are found.
```
- [x] Sanity Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added documentation to test

### CODE REVIEW

**Comment by rchengvariant** [comment]
@salyateem13 Is this for review again? I pushed with `variant-inc/actions-python@hotfix/trivy` and it failed. 
https://github.com/variant-inc/lazy-api/runs/4927910831?check_suite_focus=true

**Comment by salyateem13** [comment]
> @salyateem13 Is this for review again? I pushed with `variant-inc/actions-python@hotfix/trivy` and it failed.
> https://github.com/variant-inc/lazy-api/runs/4927910831?check_suite_focus=true

@rchengvariant Naveen created a new test-latest image... see https://github.com/variant-inc/lazy-api/runs/4927910831?check_suite_focus=true#step:6:118. This run is using v1 actions-collection

File: scripts/trivy_scan.sh [hunk 1]
```diff
@@ -35,8 +35,9 @@ fi
 echo "Printing trivy ignore file" 
 cd "$GITHUB_WORKSPACE" && cat .trivyignore
 
+
 echo "Checking for low level vulnerablities" 
-eval "trivy --exit-code 0 --severity=HIGH,MEDIUM,LOW,UNKNOWN $IMAGE"
+eval "trivy image --exit-code 0 --severity=HIGH,MEDIUM,LOW,UNKNOWN $IMAGE"
```
Comment (vibindaniel) [inline scripts/trivy_scan.sh:40]: Can you remove the severity to show everything, since the next call will hide the unfixed ones?

**Comment by rchengvariant** [comment]
Commented out CVEs in `.trivyignore` for lazy
```
# CVE-2022-23218
# CVE-2022-23219
# CVE-2021-22945
```
https://github.com/variant-inc/lazy-api/actions/runs/1751323382
[PASSED]

**Review by rchengvariant — APPROVED** [review]

**Comment by salyateem13** [comment]
Also tested negative scenario: Vulnerabilities with fixes: result is workflow failed

https://github.com/variant-inc/lazy-api/runs/4954379437?check_suite_focus=true
